sdks/python/src/honcho/peer.py

```
@@ -0,0 +1,309 @@
+
+        Returns:
+            A paginated list of Session objects this peer belongs to. Returns an empty
+            list if the peer is not a member of any sessions
+        """
+        sessions_page = self._client.workspaces.peers.sessions.list(
+            peer_id=self.id,
+            workspace_id=self.workspace_id,
+        )
+        return SyncPage(
+            sessions_page,
+            lambda session: Session(session.id, self.workspace_id, self._client),
```

Pylint (3.3.7)

[error] Using variable 'Session' before assignment

(E0601)